packages/@aws-cdk/aws-eks/test/sdk-call-integ-test-docker-app/app/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "eks-service-account-sdk-call-integ-test",
+  "dependencies": {
+    "aws-sdk": "^2.1226.0"
```

Comment:
Please commit a `package-lock.json` alongside this. Otherwise our pipeline might start failing if some sdk version breaks us.